app/services/preserved_object_handler.rb

```diff
@@ -79,28 +79,28 @@ def check_existence
           if incoming_version == pres_copy.version
             handler_results.add_result(PreservedObjectHandlerResults::VERSION_MATCHES, pres_copy.class.name)
             handler_results.add_result(PreservedObjectHandlerResults::VERSION_MATCHES, pres_object.class.name)
-            update_pc_validation_timestamps(pres_copy)
-            update_db_object(pres_copy)
-          else
-            version_comp_result = if incoming_version > pres_copy.version
-                                    PreservedObjectHandlerResults::ARG_VERSION_GREATER_THAN_DB_OBJECT
-                                  else
-                                    PreservedObjectHandlerResults::ARG_VERSION_LESS_THAN_DB_OBJECT
-                                  end
-            handler_results.add_result(version_comp_result, pres_copy.class.name)
-            handler_results.add_result(version_comp_result, pres_object.class.name)
+          elsif incoming_version > pres_copy.version
+            handler_results.add_result(PreservedObjectHandlerResults::ARG_VERSION_GREATER_THAN_DB_OBJECT, pres_copy.class.name)
+            handler_results.add_result(PreservedObjectHandlerResults::ARG_VERSION_GREATER_THAN_DB_OBJECT, pres_object.class.name)
             if moab_validation_errors.empty?
-              update_preserved_copy_version_etc(pres_copy, incoming_version, incoming_size, true)
+              update_preserved_copy_version_etc(pres_copy, incoming_version, incoming_size, false)
               update_status(pres_copy, PreservedCopy::OK_STATUS)
-              update_db_object(pres_copy)
               pres_object.current_version = incoming_version
               update_db_object(pres_object)
             else
               update_status(pres_copy, PreservedCopy::INVALID_MOAB_STATUS)
```

Comment:
would we want to `update_pc_validation_timestamps` here too, since we did check validity?  my recollection is that we update the timestamps if we did the check, regardless of the result?  i'm having a bit of trouble finding an issue/comment to cite, though that was my recollection from conversations (possibly incorrect).